promptui/select.go

```diff
@@ -25,7 +25,7 @@ func (s *Select) Run() (int, string, error) {
 }
 
 func (s *Select) innerRun(starting int, top rune) (int, string, error) {
-	stdin := readline.NewCancelableStdin()
+	stdin := readline.NewCancelableStdin(os.Stdin)
```

Comment:
Is this due to a change in the underlying package?

Should probably be noted as a dep upgrade/fix in the changelog if we're changing more than what goes into the feature.